src/hooks/Guilds/ether-swr/ens/useENSNameFromAddress.ts

```diff
@@ -1,13 +1,18 @@
-import { useProvider } from 'wagmi';
+import { useProvider, useNetwork, chainId as chains } from 'wagmi';
 import useEtherSWR from '../useEtherSWR';
 
 export default function useENSNameFromAddress(
   ensAddress?: string,
   chainId?: number
 ) {
   const provider = useProvider();
+  const { chain } = useNetwork();
+  const isLocalChain = chain?.id === chains.localhost;
+
   const { data: ensName } = useEtherSWR<string>(
-    ensAddress ? ['lookupAddress', ensAddress] : [],
+    // localhost network does not support ENS. This is a temp fix to avoid multiple errors on localhost.

```

Comment:
I have a solution for this using the function `isAvailableOnEns`. It's not the prettiest solution but makes sure that it doesn't try to resolve localhost at least.

Reply:
@Kenny-Gin1 Don't understand your intention. Do you want me to remove this change and publish your fix? I just wanted to avoid having 100 error logs per minute haha.

Reply:
Hey! Was a bit unclear. Just meant that a fix is coming hopefully 🙏